README.md

```diff
@@ -166,6 +166,8 @@ The numbers above were measured on Mac running a 4.2 GHz Intel Core i7 on Decemb
 these numbers you can see how the roundtrip is very stable and the minimal latency is now down to 17
 micro-seconds (used to be 25 micro-seconds) on this HW.
 
+**Note:** Some tools that used to be here have been moved to the [python binding](https://github.com/eclipse-cyclonedds/cyclonedds-python/tree/master/src/cyclonedds/tools).
```

Comment:
It is difficult to find a good spot for a note like this. I think it would be better to have a list of language bindings near the beginning and point out there that the python we've started migrating the tools there. That is a bit out of the scope of this PR, so perhaps the more practical approach is for you to simply remove this sentence. (But if you do want to take a shot at it, do feel free!)

Reply:
Yeah, you're right. I can't seem to find a nice place to add it, so I'm just deleting the note for now.